crates/bevy_math/src/bounding/bounded2d.rs

```diff
@@ -0,0 +1,332 @@
+use super::BoundingVolume;
+use crate::prelude::Vec2;
+
+/// A trait with methods that return 2D bounded volumes for a shape
+pub trait Bounded2d {
+    /// Get an axis-aligned bounding box for the shape with the given translation and rotation.
+    /// The rotation is in radians, clockwise, with 0 meaning no rotation.
```

Comment:
If this was degrees, I'd agree, but for radians I think it should be counterclockwise. This is how rotations work everywhere else, e.g. rotating a 2D sprite around the Z axis via transforms is counterclockwise.

This is mostly because of math conventions. I often also imagine radian angles using a unit circle (e.g. `sin(pi/2) = 1` since it's the top of the unit circle), and that is counterclockwise.

I think some kind of `Rotation` or `Angle` type in Bevy would be nice so that there'd be a consistent representation with an explicit API; that's a topic for another PR though

Reply:
I definitely want a Rotation type in another PR :)

Reply:
I made an issue for an `Angle` type: #11132

I think `Rotation2d` might be a bit too niche for now, and I'm not entirely sure if we'd even want it to be just a scalar angle. For physics and collision detection, a rotation matrix or a sine and cosine representation could be more efficient